docker/docker-compose.yml

```diff
@@ -1,29 +1,31 @@
-db:
-    image: mysql:5.7
-    container_name: mysql_mv
-    user: "mysql:mysql"
-    ports:
-        - "6603:3306"
-    volumes:
-        - ${MYSQL_DIR}:/var/lib/mysql
+version: "3.0"
```

Comment:
Tatiana, I'm curious about this "version" entry in the docker-compose.yml file. Is "3.0" meant to indicate the METviewer software version number... or the version number of this docker-compose.yml file? The version of the METviewer software is actually controlled by ${METVIEWER_DOCKER_IMAGE}. If we're introducing a new level of versioning, we'd need documentation to clarify what it actually means.

Sorry for the delay on this PR. I missed it 8 days ago.

Reply:
"version" is a Compose file version with is supported by Compose 1.13.0+

Reply:
Ah OK, thanks for clarifying. I suspect that this describes that v3 format:
https://docs.docker.com/compose/compose-file/compose-file-v3/

This seems fine then. I'll proceed with testing on my Mac using Docker Desktop version 3.5.1 and Docker Engine 20.10.7. That's what I have available.
https://docs.docker.com/compose/compose-file/compose-file-v3/